instrumentation/netty/netty-4.1/testing/src/main/java/io/opentelemetry/instrumentation/netty/v4_1/AbstractNetty41ServerTest.java

```diff
@@ -156,6 +156,11 @@ private static Object handle(HttpRequest request, URI uri, ServerEndpoint endpoi
       return response;
     }
 
+    @Override
+    public void channelInactive(ChannelHandlerContext channelHandlerContext) throws Exception {
```

Comment:
is this really needed?

Reply:
@laurit added for clarity, that this method is present. Not really needed for tests themselfs

Reply:
In my opinion having a method that just calls itself in super class is not valuable, the super method would get called anyway even if you didn't have this method

Reply:
@laurit  fixed.